### ISSUE TITLE
4 tests fail on FreeBSD

### ISSUE THREAD

**Issue by yurivict** [matplotlib/pytest-mpl#179]
[py-pytest-mpl-test.log](https://github.com/matplotlib/pytest-mpl/files/9391567/py-pytest-mpl-test.log)

Version: 0.16.1
Python-3.9
FreeBSD 13.1

**Comment by ConorMacBride** [comment]
Thanks @yurivict for testing on FreeBSD. The pytest-mpl test suite needs a specific version of FreeType installed (currently 2.6.1) so it can render fonts consistently. I can see in the log that 2.12.1 is being used (which matches https://www.freebsd.org/cgi/ports.cgi?query=^freetype2-2.12.1_2&stype=name). If you can install 2.6.1, you should be able to [get matplotlib to use that version](https://matplotlib.org/stable/devel/dependencies.html#c-libraries). Alternatively, installing matplotlib with pip will install a version compiled with 2.6.1, but I guess you want to get the tests to pass using system packages only.

**Comment by yurivict** [comment]
Yes, I can't change the version of freetype because I work in the context of the FreeBSD port.

**Comment by ConorMacBride** [comment]
I see you've added a devel/py-pytest-mpl port [here](https://github.com/freebsd/freebsd-ports/tree/3f3f7a26800cf77f964ba9e96f9800dc050305aa/devel/py-pytest-mpl), thanks! The tests under `tests/subtests` are particularly sensitive to the FreeType and should probably all be skipped. 

Although, I've just noticed another issue in the log which should be fixed first:
```
ValueError: The 'pytest_mpl.summary.html' package was not installed in a way that PackageLoader understands.
```
The patch in #180 might fix this, but I don't fully understand the root cause.

**Comment by QuLogic** [comment]
How does one update the tests here? I see that #184 added Matplotlib 3.6 hashes for subtests, but there are no hashes in `tests/baseline/hashes/` for several recent Matplotlib. I would like to at least generate hashes for more recent FreeType, but I'm not sure how best to confirm them.

**Comment by yurivict** [comment]
Now many more tests fail in 0.17.0: [see log](https://freebsd.org/~yuri/pytest-mpl-0.17.0-tests.log).
